tests/infra_test.go

```diff
@@ -301,9 +301,11 @@ var _ = Describe("Infrastructure", func() {
 			keys := getKeysFromMetrics(metrics)
 			for _, key := range keys {
 				// we don't care about the ordering of the labels
-				Expect(key).To(ContainSubstring(fmt.Sprintf("node=\"%s\"", nodeName)))
-				Expect(key).To(ContainSubstring(fmt.Sprintf("namespace=\"%s\"", vmi.Namespace)))
-				Expect(key).To(ContainSubstring(fmt.Sprintf("name=\"%s\"", vmi.Name)))
+				Expect(key).To(SatisfyAll(
+					ContainSubstring("node=\"%s\"", nodeName),
```

Comment:
What about `node="%"` so we don't have to escape ?

Reply:
I mean using golang raw string literal so we don't have to escape stuff.

Reply:
Yes, just slipped out from the past upload. Fixed now. Thanks!